apps/mull-ui/src/app/pages/messages/event-messages/event-message-list.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { ChatHeader, CustomTextInput } from '../../../components';
+import EventBullet from '../../../components/event-bullet/event-bullet';
+import './event-message-list.scss';
+
+/* eslint-disable-next-line */
+export interface EventMessageListProps {}
+
+/*TODO: Remove this and replace with query of event message channels*/
+
+const messageChannels = [
+  {
+    eventTitle: 'Clean up of Rogers Park',
+    eventPicture: 'https://i.pinimg.com/originals/c9/a1/44/c9a144b70e175c0361c6e2725677e55a.jpg',
+    eventDate: new Date(2021, 1, 1, 8, 20, 30, 1),
+    id: 1,
+  },
+  {
+    eventTitle: 'Lets do this guys!',
+    eventPicture: 'https://i.pinimg.com/originals/c9/a1/44/c9a144b70e175c0361c6e2725677e55a.jpg',
```

Comment:
is it possible to have one landscape, one portrait and one square photo to see the effect?

Reply:
sure thing ill look for one